enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -38,6 +38,20 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
         hoverBackgroundColor: "var(--mb-color-brand)",
       },
     },
+    emptyContent: {
+      icon: {
+        width: "117",
+        height: "94",
+      },
+      title: {
+        fontSize: "1.5rem",
+        textColor: "var(--mb-color-text-dark)",
+      },
+      subtitle: {
+        fontSize: "1rem",
+        textColor: "var(--mb-color-text-medium)",
+      },
```

Comment:
Curious why we have `textColor` here, as they can already override the `colors.text-dark` and `colors.text-medium` without changing this